### ISSUE TITLE
Typescript imports require fluture and fail when installed

### ISSUE THREAD

**Issue by Chase22** [Bannerets/tdl#97]
When importing anything from the tdlibs-typings module in typescript like:
`import {Chat} from "tdlib-types";`

I get the following error
`Cannot find module 'fluture' or its corresponding type declarations.`

After installing fluture using `npm install fluture` i get a whole list of error in the index.d.ts
`Future' refers to a value, but is being used as a type here. Did you mean 'typeof Future'?`

Code: https://github.com/Chase22/tdl-issue-code

**Comment by Bannerets** [comment]
I thought TypeScript shouldn't report any errors if the import is in `.d.ts` and not `.ts`. I've used `tdl` with TypeScript and didn't have any errors.

I cloned your repository and also cannot reproduce the issue, with or without fluture. My `tsc` is version 4.1.3.

Maybe `skipLibCheck` is set to false in your `tsconfig.json`? `tsc --init` generates `tsconfig.json` with the option set to true. It is also `true` in your https://github.com/Chase22/tdl-issue-code.
